content/admin/packages/quickstart-for-configuring-your-minio-storage-bucket-for-github-packages.md

```diff
@@ -24,7 +24,6 @@ This quickstart shows you how to set up MinIO using Docker for use with {% data
 | MinIO mode | Optimized for | Storage infrastructure required |
 |----|----|----|
 | Standalone MinIO (on a single host) | Fast setup |  N/A |
-| MinIO as a NAS gateway |  NAS (Network-attached storage)| NAS devices |
 | Clustered MinIO (also called Distributed MinIO)|  Data security | Storage servers running in a cluster |
```

Comment:
Should we have similar table (which talks about different storage modes) for Actions page?